contrib/example-dynamic-root-menu.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+#
+# Example usage (~/.config/labwc/menu.xml):
+#
+#     <?xml version="1.0" encoding="utf-8"?>
+#     <openbox_menu>
+#       <menu id="root-menu" label="" execute="example-dynamic-root-menu.sh"/>
+#     </openbox_menu>
+
+terminal="foot"
+
+printf '%b\n' '<openbox_pipe_menu>
+  <item label="Web Browser">
+    <action name="Execute"><command>firefox</command></action>
+  </item>
+  <item label="Terminal">
+    <action name="Execute" command="foot" />
```

Comment:
I suppose this could be
```
<action name="Execute" command="'"$terminal"'" />
```
(That is `command="` `'` `"` `$terminal` `"` `'` `" />`)